metadata/org.jboss.logging/jboss-logging/3.5.0.Final/reflect-config.json

```diff
@@ -0,0 +1,109 @@
+[
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.LoggerProviders"
+    },
+    "name": "ch.qos.logback.classic.Logger"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "java.io.FilePermission"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "java.lang.RuntimePermission"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "java.net.NetPermission"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "java.net.SocketPermission"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "java.net.URLPermission",
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": [
+          "java.lang.String",
+          "java.lang.String"
+        ]
+      }
+    ]
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "java.security.AllPermission"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "java.security.SecurityPermission"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "java.util.PropertyPermission"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.Slf4jLoggerProvider"
+    },
+    "name": "javax.smartcardio.CardPermission"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.LoggerProviders"
+    },
+    "name": "org.apache.log4j.LogManager"
+  },
+  {
+    "condition": {
+      "typeReachable": "org.jboss.logging.LoggerProviders"
+    },
+    "name": "org.apache.logging.log4j.Logger"
+  },
```

Comment:
I don't think hints should be added for Log4j2 classes. It will introduce another cause of the problem that https://github.com/oracle/graalvm-reachability-metadata/pull/108 fixes.

Reply:
You're right, I'll remove them.